index.js

```diff
@@ -91,7 +91,8 @@ function serveStatic(root, options) {
       path = ''
     }
 
-    // create send stream
+    // create send stream (etag computation and other options are taken care of
```

Comment:
Why call out etag specifically in this comment and leave all others as "and other options"? Would not someone be coming here just as likely look for how lastModified works just like you did for etag? The comment basically seems overly biased and only helps people with the single case, just like the readme documentation did. Can you update the comment such that it is helpful to anyone and not just the etag seekers?